### PR TITLE
melmatcheq.lv2: init at 0.1

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/melmatcheq.lv2/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    xorg.libX11 xorgproto cairo lv2
+  ];
+
+  installFlags = [ "INSTALL_DIR=$(out)/lib/lv2" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/brummer10/MelMatchEQ.lv2";
+    description = "a profiling EQ using a 26 step Mel Frequency Band";
+    maintainers = [ maintainers.magnetophon ];
+    license = licenses.gpl3;
```
Comment (sikmir) [inline pkgs/applications/audio/melmatcheq.lv2/default.nix:25]: It's `gpl3` according to [LICENSE](https://github.com/brummer10/MelMatchEQ.lv2/blob/master/LICENSE), but `gpl2Plus` according to [sources](https://github.com/brummer10/MelMatchEQ.lv2/blob/3d91ab9eefe130986d2e6b7700c14726d85d044c/plugin/gx_melmatcheq.h#L6)...

Comment (magnetophon) [reply pkgs/applications/audio/melmatcheq.lv2/default.nix:25]: Good catch, I opened [an issue](https://github.com/brummer10/MelMatchEQ.lv2/issues/1).

Comment (magnetophon) [reply pkgs/applications/audio/melmatcheq.lv2/default.nix]: Fixed upstream, and PR adapted accordingly.

File: pkgs/applications/audio/melmatcheq.lv2/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  buildInputs = [
+    xorg.libX11 xorgproto cairo lv2
+  ];
+
+  installFlags = [ "INSTALL_DIR=$(out)/lib/lv2" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/brummer10/MelMatchEQ.lv2";
+    description = "a profiling EQ using a 26 step Mel Frequency Band";
+    maintainers = [ maintainers.magnetophon ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/melmatcheq.lv2/default.nix:24]:
```suggestion
    maintainers = with maintainers; [ magnetophon ];
```

**Comment by SuperSandro2000** [comment]
And why labeled ofborg this as 0 builds for darwin and linux?

File: pkgs/applications/audio/melmatcheq.lv2/default.nix (hunk 2 above)
Comment (magnetophon) [reply pkgs/applications/audio/melmatcheq.lv2/default.nix:24]: Thanks for the review.
May I ask why you propose this change?
I can see the point of ``with`` as part of a [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) strategy, but I don't see the point in this case, as there is only one maintainer.

Comment (SuperSandro2000) [reply pkgs/applications/audio/melmatcheq.lv2/default.nix]: If there are two in the future you don't need to change it.

**Comment by magnetophon** [comment]
@SuperSandro2000 Maybe because I didn't have meta.platform defined? Seems OK now, right?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102743` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/applications/audio/melmatcheq.lv2/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "brummer10";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s805jgb9msxfq9047s7pxrngizb00w8sm4z94iii80ba65rd20x";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/melmatcheq.lv2/default.nix:14]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
```

File: pkgs/applications/audio/melmatcheq.lv2/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkgconfig }:
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/melmatcheq.lv2/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkg-config }:
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102743` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/applications/audio/melmatcheq.lv2/default.nix (hunk 4 above)
Comment (magnetophon) [reply pkgs/applications/audio/melmatcheq.lv2/default.nix]: Ah, yes, thanks.
Out of curiosity: is there a reason to not do a tree-wide replace of ``pkgconfig`` for ``pkg-config``, except for the alias?
I usually start new pkgs by copying a similar existing one and adapting it, so this kind of error is easy to creep in.
I can't imagine I'm the only one doing this.

**Pull request merged**